tools-local/tasks/FinalizeBuild.cs

```diff
@@ -97,6 +97,8 @@ public override bool Execute()
                     CopyBlobs($"{Channel}/Installers/{SharedFrameworkNugetVersion}", $"{Channel}/Installers/Latest/");
 
                     CopyBlobs($"{Channel}/Installers/{SharedHostNugetVersion}", $"{Channel}/Installers/Latest/");
+                    
+                    CopyBlobs($"{Channel}/Checksums/{SharedFrameworkNugetVersion}", $"{Channel}/Checksums/Latest/");
```

Comment:
The checksums have to go to a separate blob storage.  It doesn't make sense to put the checksums and the bits in the same place.  If that one place gets compromised, the checksums are no good.